dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/python.py

```diff
@@ -59,44 +59,46 @@ def foo():
         "raw_script",
     }
 
+    ext: set = {".py"}
+    ext_attr: str = "_raw_script"
+
     def __init__(
         self, name: str, definition: Union[str, types.FunctionType], *args, **kwargs
     ):
+        self._raw_script = self.raw_script(definition)
         super().__init__(name, TaskType.PYTHON, *args, **kwargs)
-        self.definition = definition
```

Comment:
we should not remove the statement `self.definition = definition`, I think we should better use 

```python
    # For extension record
    ext: set = {".py"}
    ext_attr: str = "_definition"

    def __init__(
         self, name: str, definition: Union[str, types.FunctionType], *args, **kwargs
     ):
         self._raw_script = self.raw_script(definition)
         super().__init__(name, TaskType.PYTHON, *args, **kwargs)
         self._definition = definition
```

Reply:
And keep  other setting in the old style